### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by jesonann** [getlantern/lantern-forum-cn#517] [General]
使用体验不佳，打开网页非常慢，申请退款，邮件发了，然后服务器拒收了，不知道什么意思。申请退款 ，麻烦专人回复一下，急

**Comment by jesonann** [comment]
邮箱账号：goldlipie@163.com

**Comment by MaxMKMax** [comment]
@jesonann 您好, 好的我帮您提交退款. 但需要您提供我只付单据. 谢谢.
